src/www/services_dhcp.php

```diff
@@ -990,17 +990,15 @@ function removeRow() {
                           <input type="checkbox" value="yes" name="netboot" id="netboot" <?=!empty($pconfig['netboot']) ? " checked=\"checked\"" : ""; ?> />
                           <strong><?=gettext("Enables network booting.");?></strong>
                           <br/>
-                          <?=gettext("Enter the IP of the"); ?> <b><?=gettext("next-server"); ?></b>
-                          <input name="nextserver" type="text" id="nextserver" value="<?=$pconfig['nextserver'];?>" /><br />
-                          <?=gettext("and the default bios filename");?>
-                          <input name="filename" type="text" id="filename" value="<?=$pconfig['filename'];?>" /><br />
-                          <?=gettext("and the UEFI 32bit filename  ");?>
-                          <input name="filename32" type="text" id="filename32" value="<?=$pconfig['filename32'];?>" /><br />
-                          <?=gettext("and the UEFI 64bit filename  ");?>
-                          <input name="filename64" type="text" id="filename64" value="<?=$pconfig['filename64'];?>" /><br />
+                          <?=sprintf(gettext("Enter the IP of the next-server %s and the default bios filename %s and the UEFI 32bit filename %s and the UEFI 64bit filename %s"),
+                                    '<input name="nextserver" type="text" id="nextserver" value="' . $pconfig['nextserver'] . '" /><br />',
+                                    '<input name="filename" type="text" id="filename" value="' . $pconfig['filename'] . '" /><br />',
+                                    '<input name="filename32" type="text" id="filename32" value="' . $pconfig['filename32'] . '" /><br />',
+                                    '<input name="filename64" type="text" id="filename64" value="' . $pconfig['filename64'] . '" /><br />'
+                                    ) ?>
```

Comment:
@fabianfrz I would rather leave the input tags out of the parsed string data, but that's probably just a matter of style preference. 
Maybe @fichtner disagrees with me here, so I will leave this pull request for him :)
As always, thanks for you help!